File: Python/problem9.py
given_sentence = "Good days give happiness, bad days give experience, worst days give lessons, and the best days give memories."

def exaggerate(sentence):
    replacements = {
        "good": "great",
        "bad": "terrible",
        "like": "love",
        "dislike": "hate"
    }

    all_words = sentence.split()
    for word in all_words:
        word = word.replace(",", "")
    return all_words

print(exaggerate(given_sentence))

File: JavaScript/problem9.js
const sentence = 'Good days give happiness, bad days give experience, the days I dislike give lessons, and the days I love give memories.'

function exaggerate(sentence) {
    wordArr = sentence.split(" ")
    for (let i = 0; i < wordArr.length; i++) {
        if (wordArr[i] == "Good") {
            wordArr[i] = "Great"
        }
        if (wordArr[i] == "bad") {
            wordArr[i] = "terrible"
        }
      if (wordArr[i] == "like") {
            wordArr[i] = "love"
      }
      if (wordArr[i] == "dislike") {
            wordArr[i] = "hate"
      }
    }
    return wordArr.join(" ")
}

function exaggerate2(sentence) {
    return sentence
            .replace(/good/gi, "Great")
            .replace(/dislike/gi, "hate")
            .replace(/bad/gi, "terrible")
            .replace(/like/gi, "love")
}

console.log(exaggerate(sentence))
console.log(exaggerate2(sentence))